Userland/Libraries/LibC/sys/wait.cpp

```diff
@@ -64,7 +64,7 @@ pid_t waitpid(pid_t waitee, int* wstatus, int options)
             *wstatus = siginfo.si_status << 8 | 0x7f;
             break;
         case CLD_CONTINUED:
-            *wstatus = 0;
+            *wstatus = 0xffff;
```

Comment:
I'm going to guess that this should be in the second commit then?

Reply:
I don't think so. We set `wstatus` to 0xffff so that `WIFCONTINUED` works by testing against that value. POSIX does not specify what value we put in `wstatus`.

Reply:
Ah, I see, I haven't made the connection there. Although, it would be nice to explicitly mention that in the commit message (something like "Since we now define a value for the continue state, make sure to set it as well"), and maybe have some defines for those magic numbers down the line.